### ISSUE TITLE
How do I train on my own dataset?

### ISSUE THREAD

**Issue by soheildousti** [bytedance/LatentSync#47]
Hello
I worked on the project https://github.com/Rudrabha/Wav2Lip. And I encountered the problem mentioned in the article and loss = 0.69 did not decrease and no matter how much I changed it, I still could not reduce it. Now I want to proceed with the codes of this project. Can anyone help me? Tell me step by step what to do? I want to try it on my dataset in Persian language and my first goal is to detect the sync of audio and video and my next goal is to produce a video in the target language. Please help me.
